src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -84,8 +87,10 @@ def is_installed(cls):
             HAVE_MS5 = True
         except ImportError:
             HAVE_MS5 = False
+            mountainsort5 = None
 
         if HAVE_MS5:
+            assert mountainsort5
```

Comment:
```suggestion
```

Reply:
Also this is for the linter?

Reply:
yes, according to pyright you can't access mountainsort5 if it is possibly None, and it's not smart enough in this case to know that it cannot be None. So that's why the assert is needed.